test/integration/targets/setup_mysql8/handlers/main.yml

```diff
@@ -0,0 +1,10 @@
+- name: remove repo
+  yum:
+    name: mysql80-community-release
+    state: absent
+  listen: cleanup mysql8
+
+- name: remove mysql
+  yum:
+    name: '{{ mysql_package_name }}'
```

Comment:
```suggestion
    name: '{{ mysql_package_name }}'
    state: absent
```

Reply:
and would be cool to stop the services before removing packages, FS objects, etc

Reply:
Good idea. I'll add this.